backend/question-service/src/app.ts

```diff
@@ -3,12 +3,17 @@ import router from "./routes";
 import bodyParser from "body-parser";
 import cors from "./middleware/cors";
 import HttpStatusCode from "./lib/enums/HttpStatusCode";
-import dotenv from "dotenv";
+import PinoHttp from "pino-http";
+import logger from "./lib/utils/logger";
 import { authMiddleware } from "./middleware/auth";
+import dotenv from "dotenv";
 
 dotenv.config();
 
-const app: Express = express();
+const app = express();
+const expressLogger = PinoHttp({ logger });
+
+app.use(expressLogger);
```

Comment:
Can I ask for this logger, if we are in production, where will the message be logged to? Also, are we going to log the entire request even if it is 2xx? I haven't set up any logger in the API so far because I don't think we will use any backlog services like Splunk. Can the API request and response be automatically logged to the AWS logging monitoring tool?

Reply:
After we deploy to aws, all console messages will be pushed to aws cloudwatch automatically. As for the log level, it will change based on the built environment, currently it's set as `debug` for `development` and `info` for `production` builds, so if the `logger.ts` is properly configured, the logger should suppress any log that's lower priority than `info` in `production`.

Reply:
I see, then I am fine with the logger set up we have right now :) Thanks for clarifying!